app/js/app.jsx

```diff
@@ -29,6 +31,9 @@ const rvMuiTheme = getMuiTheme({
 
 let store = createStore(analyticsApp)
 
+// Check if user is logged in and add headers to new ajax calls
+Auth.init()
+
```

Comment:
Not very important but maybe this should go before `injectTapEventPlugin()`, so the first thing that we do is checking for credentials, or is there a reason to have it here?